stac_pydantic/api/search.py

```diff
@@ -108,9 +108,9 @@ def validate_datetime(cls, value: str) -> str:
                 "Invalid datetime range. Too many values. Must match format: {begin_date}/{end_date}"
             )
 
-        # If there is only one date, insert a None for the start date
+        # If there is only one date, duplicate to use for both start and end dates
         if len(values) == 1:
-            values.insert(0, None)
+            values = [values[0], values[0]]
```

Comment:
I went for a more `verbose` way (ref: https://github.com/stac-utils/stac-pydantic/pull/135#pullrequestreview-2054397883)